sgf-viewer/src/components/subcomponents/Contents.svelte

```diff
@@ -0,0 +1,182 @@
+<script lang="ts">
+    import { onMount, onDestroy } from "svelte";
+    import { pages } from "../../content";
+    import { fade } from "svelte/transition";
```

Comment:
```suggestion
    import { onMount, onDestroy } from "svelte";
    import { fade } from "svelte/transition";
    
    import { pages } from "../../content";
```
(just grouping together external-library imports vs. relative file imports)